fix: clean self node's cluster commit when leave cluster (#12843)
emqx/emqx: changes/ce/fix-12843.en.md

> Fixed cluster_rpc_commit tnx_id was not properly cleanup after 'cluster leave' on replicant nodes,

Reviewer: ```suggestion
Fixed `cluster_rpc_commit` transaction ID cleanup procedure after `cluster leave` on replicant nodes.
```

> Fixed cluster_rpc_commit tnx_id was not properly cleanup after 'cluster leave' on replicator nodes,

Reviewer 2: ```suggestion
Fixed cluster_rpc_commit tnx_id was not properly cleanup after 'cluster leave' on replicant nodes,
```

Author: Done

> The tnx_id of the core node will be deleted before, resulting in the failure of the core node update configuration.

Reviewer: ```suggestion
Previously, the transaction id of the core node would be deleted prematurely, blocking configuration updates on the core node.
```
